packages/autofix/lib/rules/no-unused-vars.js

```diff
@@ -25,6 +98,36 @@ module.exports = ruleComposer.mapReports(
             const grand = parent.parent;
 
             switch (parent.type) {
+                case "FunctionExpression":
+                case "FunctionDeclaration":
+                case "ArrowFunctionExpression":
+
+                    // Don't autofix unused functions, it's likely a mistake
+                    if (parent.id === node) {
+                        return null;
+                    }
+
+                    switch (config.args) {
+                        case "all": {
+                            const prefix = getIgnoredArgPrefix(config);
+
+                            if (prefix === null) {
+                                return null;
+                            }
+
+                            const i = node.range[0];
+
+                            return fixer.replaceTextRange([i, i], prefix);
```

Comment:
```suggestion
                            return fixer.insertTextBefore(node, prefix);
```

nits: seems it can be simplified to `fixer.insertTextBefore()`.  refs: https://eslint.org/docs/developer-guide/working-with-rules#applying-fixes

Reply:
Ah, thanks, I didn't notice that method when I was reading the docs. Applied!